container.go

```diff
@@ -624,3 +645,64 @@ func (container *container) unregisterCallback() error {
 
 	return nil
 }
+
+// Modifies the System by sending a request to HCS
+func (container *container) Modify(config *ResourceModificationRequestResponse) error {
+	operation := "Modify"
+	title := "HCSShim::Container::" + operation
+
+	if container.handle == 0 {
+		return makeContainerError(container, operation, "", ErrAlreadyClosed)
+	}
+
+	container.handleLock.RLock()
+	defer container.handleLock.RUnlock()
+
+	requestJSON, err := json.Marshal(config)
+	if err != nil {
+		return err
+	}
+
+	var (
+		resultp *uint16
+	)
+
+	requestString := string(requestJSON)
+	logrus.Debugf(title+" id=%s request=%s", container.id, requestString)
+
+	err = hcsModifyComputeSystem(container.handle, requestString, &resultp)
+	err = processHcsResult(err, resultp)
+	if err != nil {
+		return makeContainerError(container, operation, "", err)
+	}
+	logrus.Debugf(title+" succeeded id=%s", container.id)
+	return nil
+}
+
+// ModifyContainer corresponding to the container id, by sending a request
+func ModifyContainer(id string, request *ResourceModificationRequestResponse) error {
```

Comment:
Will this return error on RS1? Will it be propagated to libnetwork? This might break docker daemon from working on RS1. Can you please try this?